src/Sulu/Bundle/SecurityBundle/Behat/SecurityContext.php

```diff
@@ -127,7 +153,24 @@ public function iAmLoggedInAsAnAdministrator()
         $this->getSession()->wait(5000, "document.querySelector('.navigation')");
     }
 
-    private function getOrCreateRole($name, $system)
+    /**
+     * @Given I am editing the permission of a user with username :username
+     */
+    public function iAmEditingThePermissionsOfAUser($username)
+    {
+        /** @var User $user */
+        $user = $this->getEntityManager()
+            ->getRepository('SuluSecurityBundle:User')->findOneBy(
+                ['username' => $username]
+            );
+
+        $this->visitPath('/admin/#contacts/contacts/edit:' . $user->getContact()->getId() . '/permissions');
+        $this->getSession()->wait(5000, '$("#permissions-grid").length');
+        sleep(1); // wait one more second to avoid flaky tests
```

Comment:
Why is that necessary? Can't that be solved in a more elegant way? Isn't there an event, which can be listened?